engine/verification/verifier/engine_test.go

```diff
@@ -116,20 +114,6 @@ func (suite *VerifierEngineTestSuite) TestNewEngine() *verifier.Engine {
 
 }
 
-func (suite *VerifierEngineTestSuite) TestInvalidSender() {
```

Comment:
Verifier engine is only receiving input through `ProcessLocal` from other engines of the same verification node, hence, this test is no longer needed.